### PR TITLE
Remove expanding group error style

### PR DESCRIPTION
**Pull request by cehsu**
Connects to department-of-veterans-affairs/vets.gov-team#3686

screen shot 2017-07-25 at 11 23 57 am

### CODE REVIEW

File: src/sass/modules/_m-form-elements.scss [hunk 1]
```diff
@@ -142,13 +142,6 @@ button.form-button-disabled {
 }
 
 .form-expanding-group {
-  .usa-input-error {
```
Comment (cvalarida) [inline src/sass/modules/_m-form-elements.scss]: Oh, that's...really bizarre. Why was this here in the first place?

**Review by cvalarida — APPROVED** [review]

File: src/sass/modules/_m-form-elements.scss (hunk 1 above)
Comment (jbalboni) [inline src/sass/modules/_m-form-elements.scss]: I must have added it a while back so that the bars lined up underneath a group. Not sure why it would be different now.

**Comment by jbalboni** [comment]
@cehsu @cvalarida This is why the styles were there:

screen shot 2017-07-27 at 9 37 05 am

**Comment by cehsu** [comment]
@jbalboni @cvalarida Is the desired behavior to remove the blue line?

**Comment by jbalboni** [comment]
I think the desired behavior is to have the red line to the right of the blue line, but maybe @goldenmeanie can weigh in.

**Comment by goldenmeanie** [comment]
Yes, original intent was to the right of the blue bar.

Can see a couple ways forward here:
1. To right of bar, fields stay lined up
2. To right of bar, fields bumped over to maintain spacing with alert highlight
3. Total eclipse of the blue bar, effectively turning part of it red

**Comment by cehsu** [comment]
Is this open to a vote? If so, I vote option 3. If not, that's cool too.

**Comment by cvalarida** [comment]
I think the way we had it before is fine, but if there is no blue line, the red line needs to line up with the other red lines. I'd need to look at it more closely, but I think we can manage this with some css-fu. (So option 1.)

Basically, the trouble is that the fields inside an `expandUnder` get the blue line, but fields inside a `hideIf` do not, but the shifted red line applies to both.

**Comment by cehsu** [comment]
@cvalarida so would hideIf need to add a class, and then if an element has that class, we would unset the styles added in the currently deleted code (which would be restored)?

**Comment by jbalboni** [comment]
I don't think `form-expanding-group` is used for `hideIf` fields.

**Comment by cehsu** [comment]
Thanks for the clarification @jbalboni, so the desired behavior is to keep things the same? Should we close this PR?

**Comment by jbalboni** [comment]
@cehsu I think what @cvalarida created the original issue for is still a problem. I think you should look into that and figure out the cause, and/or a way to fix it while keeping the red par position inside expanded groups.

**Comment by cehsu** [comment]
It looks like we should be using expandUnderClassNames for this instead?

**Comment by cehsu** [comment]
Is this the desired behavior?
### Not expanded
screen shot 2017-07-27 at 3 14 34 pm

### Expanded
screen shot 2017-07-27 at 3 13 33 pm

**Comment by jbalboni** [comment]
@cehsu Yeah, that's how it was intended to look before. If this is what the normal behavior should be, I don't want to make us have to set `expandUnderClasses` everywhere to make it happen, though.

**Comment by cehsu** [comment]
#### Not expanded
screen shot 2017-07-28 at 10 54 19 am

#### Expanded
screen shot 2017-07-28 at 10 54 38 am

#### Expanded with `schemaform-expandUnder-indent` class
screen shot 2017-07-28 at 10 53 41 am

File: src/sass/modules/_m-form-elements.scss [hunk 2]
```diff
@@ -141,11 +141,17 @@ button.form-button-disabled {
...
 
-.form-expanding-group {
+.form-expanding-group > span {
   .usa-input-error {
     padding-left: calc(1.5rem - 6px);
     right: calc(1.9rem - 6px);
   }
+  .schemaform-expandUnder-indent {
+    .usa-input-error {
+      padding-left: 15px;
```
Comment (jbalboni) [inline src/sass/modules/_m-form-elements.scss]: Can we use the original units for this? `rem`, I think.

**Review by jbalboni — APPROVED** [review]
LGTM, had one nitpick

**Comment by cehsu** [comment]
Thanks @jbalboni, is the updated code what you had in mind?

**Comment by jbalboni** [comment]
@cehsu I'm not sure you need the calc. Shouldn't those values match the default from USWDS?

**Comment by cehsu** [comment]
Thanks @jbalboni, you're right. Change made.
